styles/styles.css

```diff
@@ -0,0 +1,534 @@
+
+/*Universal Stylings*/
+* {
+  box-sizing: border-box;
+  font-family: 'Merriweather', sans-serif;
+}
+
+/*Header*/
+.header {
+  background-image: linear-gradient(90deg, rgba(0,0,57,1) 0%, rgba(9,11,121,1) 45%, rgba(136,133,242,1) 100%);
+  background-image: url(../images/header-image.jpg);
+  background-size: 100%;
+  background-repeat: no-repeat;
+  margin: 0;
+  padding: 6%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.title > h1 {
+  font-style: italic;
+  font-size: 2em;
+  color: #418c27;
+}
+
+.title > img {
+  width: 85%;
+
+}
+
+.checkout {
+  display: grid;
+  justify-items: flex-end;
+  grid-gap: 1em;
+}
+
+.checkout > p {
+  color: #ffffff;
+  margin: 0;
+  align-self: end;
+}
+
+.checkout > a {
+  text-decoration: none;
+  color: #ffffff;
+  background-color: #81b741;
+  padding: 5% 20%;
+  border-radius: .2em;
+  grid-column: 1 / 3;
+  align-self: flex-start;
+}
+
+.cart {
+  width: 20%;
+  align-self: end;
+}
+
+
+/* Header: Naviagation */
+.top-navigation {
+  background-color: #e8e8e8;
+  color: #1c356d;
+  padding: .5% 12% 3%;
+  height: 6em;
+  margin: auto;
+}
+
+.nav-links{
+  font-weight: bold;
+  font-size: 1.5em;
+}
+
+.nav-links > a {
+  text-decoration: none;
+  color: #1c356d;
+  font-weight: bold;
+  padding: 0 1%;
+}
+
+.top-navigation> a:first-child {
+  padding: 0 1% 0 0;
+}
+
+
+/*Body*/
+
+.main {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
```

Comment:
grid-template-columns: repeat( 4, minmax(25%, 1fr) );

Try something like the above to ensure they stay even sized.